shared/types.ts

```diff
@@ -2,6 +2,7 @@ import type { InferSelectModel } from 'drizzle-orm'
 import type { events, news, users } from '~/server/db/schema'
 
 export type User = InferSelectModel<typeof users>
+export type ImportedUser = Omit<User, 'firebaseId' | 'id'>
```

Comment:
Is this typed used anywhere other than on front end

Reply:
Nope, should I add it somewhere else?

Reply:
Yeah i think can just inline